rpm/libtasn1.spec

```diff
@@ -30,7 +36,6 @@ This package contains tools using the libtasn library.
 
 %package devel
 Summary:    Files for development of applications which will use libtasn1
-Group:      Development/Libraries
 Requires:   %{name} = %{version}-%{release}
 Requires(post): /sbin/install-info
```

Comment:
These install-info dependencies in devel package are a leftover from the past. They should be removed, I think.